docs/_Docs_Permissions.md

```diff
@@ -4,74 +4,76 @@ section: Docs
+Note: Currently `Arbitration` role grants permission for no functions. It is a placeholder for the dispute resolution system, to be implemented in later releases.
 
-## Definitions
+## Domain permission transitivity
+*Note: Domains are currently restricted to one level below the root domain. This restriction will be removed after release.*
 
-Roles are defined within `ColonyDataTypes.sol`, and initialized for functions in `ColonyAuthority.sol`
+Domain permissions extend from the root domain. Permissions held in a domain are held in all child sub-domains, but not in parent domains.
 
-### Administration
-```
-makeTask
-addPayment
-setPaymentRecipient
-setPaymentDomain
-setPaymentSkill
-setPaymentPayout
-finalizePayment
-```
+As an example, consider this tree of domains in a colony (using domainIds as identifiers):
 
-### Funding
 ```
-moveFundsBetweenPots
+      1
+   /  |  \
+  2   4   6
+ / \  
+3   5
 ```
 
-### Architecture
+Authority in domain `2` to call a permissioned function is valid in domains `3` and `5`, but not `6`. Authority in domain `1` to call a permissioned function is valid in all subdomains.
+
+## Using permissioned functions
+Permissioned functions check two arguments, which are by convention the first and second ones expected in all permissioned functions:
+
+`_permissionDomainId`: The domain that gives the caller the authority to execute an action
+`_childSkillIndex`: an index that specifies where to find the domain in which the action occurs.
 
+New domains are given a unique skillId upon creation, so a colony with the following domain structure
 ```
-addDomain
-setArchitectureRole
-setFundingRole
-setAdministrationRole
+      1
+   /  |  \
+  2   4   6
+ / \  
+3   5
+```
+might have local skillIds assigned as:
+```
+       142
+    /   |   \
+  147  254  696
+  / \  
+159  307
 ```
-Note: Within `ColonyAuthority.sol` you will see this role implemented as `ARCHITECTURE_ROLE` and `ARCHITECTURE_SUBDOMAIN_ROLE`. This is in order to prohibit an architect from modifying the domain in which the role was given (which would allow them to, for example, remove their co-architect's role). Architects may alter permissions only in sub-domains only.
 
-### Arbitration
+In this example,
+Skill `142` has children: `[147, 159, 254, 307, 696]`
+Skill `147` has children: `[159, 307]`
+Skill `254` has children: `[]`
 
-Currently this role grants permission for no functions. It is a placeholder for the dispute resolution system, to be implemented in later releases.
+If a user with "Admininstration" authority in domain `2` wants to finalize a payment in domain `5`, they would call:
```

Comment:
I hope you approve of me changing the example function, @elenadimitrova. I think it's a little confusing to make the first example of permissioned functions a function that alters permissions :stuck_out_tongue: